api/src/main/resources/workbench-api.yaml

```diff
@@ -2341,25 +2341,58 @@ paths:
           description: The updated Institution.
           schema:
             "$ref": "#/definitions/Institution"
-  "/v1/institutions/{shortName}/checkEmail/{email}/":
+  "/v1/institutions/{shortName}/checkEmail":
     parameters:
     - in: path
       name: shortName
       type: string
       description: 'The short name / key of the institution to check, e.g. "VUMC"'
       required: true
-    - in: path
-      name: email
-      type: string
-      description: Institutional contact email address to check.
-      required: true
-    get:
+    post:
       tags:
         - institution
       operationId: checkEmail
       description: Checks whether the given email address is a verified member of an institution.
       x-aou-note: This endpoint is publicly-accessible and called by clients during user registration.
       security: []
+      parameters:
+      - in: body
+        name: contactEmail
+        description: Institutional contact email address to check.
+        required: true
+        schema:
+          type: object
```

Comment:
I would prefer to use `type: string` here but something gets lost in translation.  The Typescript wrapper adds double quotes to the string it passes to the endpoint (so, a valid JSON String) but the Java endpoint here treats the JSON String as a **Java** String without parsing the quotes away.

An alternative solution would be to manually strip the quotes.  But that feels too reliant on Swagger implementation so I'd be concerned that this might change with an upgrade.

Reply:
Yeah, I'm not too surprised there's weirdness in trying to have a bare string for a POST body. I think what you've done is the best approach, just go with the flow and make it a simple JSON object.

[opt] The vast majority of our `in: body` definitions have a named `schema: $ref`. For consistency & explicitness, I'd recommend creating a CheckEmailRequest object here rather than inlining the schema.

Reply:
ok